docs/en/stack/ml/anomaly-detection/ootb-ml-jobs.asciidoc

```diff
@@ -0,0 +1,931 @@
+[role="xpack"]
+[[ootb-ml-jobs]]
+== Prebuilt {anomaly-jobs}
+
+{anomaly-jobs-cap} contain the configuration information and metadata necessary 
+to perform an analytics task. {kib} can recognize certain types of data and 
+provide specialized wizards for that context. This page lists the available 
+prebuilt {anomaly-jobs} that you can use via {kib}.
+
+
+[float]
+[[ootb-ml-jobs-apache]]
+=== Apache
+
+tag::apache-jobs[]
```

Comment:
These tags are appearing in the preview, so I think we have to comment them like this (see https://github.com/elastic/elasticsearch/blob/7.6/docs/reference/release-notes/highlights-7.6.0.asciidoc for working example):
```suggestion
// tag::apache-jobs[]
```